pkg/account/session/session.go

```diff
@@ -50,6 +51,7 @@ func (s *Session) authCookie() *cookie.Cookie {
 		Name:     authCookieName,
 		Path:     "/",
 		HTTPOnly: true,
+		SameSite: http.SameSiteLaxMode,
```

Comment:
So shouldn't this be `http.SameSiteStrictMode` then?

Reply:
No, because if you navigate from the `nam1` Console to the `eu1` Account app in order to edit your account settings, you don't want to be greeted with a login screen because your browser doesn't want to send the cookie.